packages/koa-shopify-auth/package.json

```diff
@@ -32,10 +32,10 @@
   "devDependencies": {
     "@shopify/jest-dom-mocks": "^2.8.0",
     "@shopify/jest-koa-mocks": "^2.1.5",
-    "@types/koa": "^2.0.44",
+    "@types/koa": "^2.0.50",
     "@types/koa-compose": "*",
     "@types/safe-compare": "^1.1.0",
-    "koa": "^2.5.0"
+    "koa": "^2.8.2"
```

Comment:
Just a heads up that you don't generally need to update dependencies in this way, using `yarn-deduplicate` can get you down to a single set of installed dependencies without forcing more restrictive version ranges